### PR TITLE
Add: heart rate data for gpx

### PR DESCRIPTION
**Pull request by yihong0618**
You can test from my running data https://github.com/yihong0618/blog/tree/master/data/activities
And hint: 
you can change the date to a future date like 2020-08-02 in gpx and upload to strava or runtastic to see if it is right or not

### CODE REVIEW

**Comment by yasoob** [comment]
I will test this later today and hopefully merge it in. Looks good on a cursory look.

**Comment by yihong0618** [comment]
@yasoob 
Add single or mutiple NRC files support, also add mutiple folders support.
See:
image
image

File: nrc_exporter.py [hunk 1]
```diff
@@ -371,29 +373,54 @@ def generate_gpx(title, latitude_data, longitude_data, elevation_data):
...
-                counter += 1
-            gpx_segment.points.append(
-                gpxpy.gpx.GPXTrackPoint(
-                    lat["value"],
-                    lon["value"],
-                    time=d_time,
-                    elevation=elevation_data[counter]["value"],
-                )
-            )
+        points_dict_list.append({"latitude": lat["value"], "longitude": lon["value"], "start_time":  lat["start_epoch_ms"], "time": datetime.datetime.utcfromtimestamp(lat["start_epoch_ms"] / 1000)})
```
Comment (yasoob) [inline nrc_exporter.py:403]: Can you please split this to multiple lines? Something like this:

```python
points_dict_list.append({
    "latitude": lat["value"], 
    "longitude": lon["value"], 
    "start_time":  lat["start_epoch_ms"], 
    "time": datetime.datetime.utcfromtimestamp(lat["start_epoch_ms"] / 1000)
})
```

File: nrc_exporter.py [hunk 2]
```diff
@@ -490,7 +524,7 @@ def arg_parser():
     ap.add_argument("-v", "--verbose", action="store_true", help="print verbose output")
     ap.add_argument("-t", "--token", help="your nrc token", required=False)
     ap.add_argument(
-        "-i", "--input", help="A directory containing NRC activities in JSON format"
+        "-i", "--input", nargs='+', help="A directory containing NRC activities in JSON format or NRC JSON files"
```
Comment (yasoob) [inline nrc_exporter.py:527]: Please add more to the help message regarding the fact that people can also specify multiple folders containing the JSON files. Maybe we can change this to: "A directory or directories containing NRC activities in JSON format. You can also provide individual NRC JSON files"

Thoughts?

Comment (yihong0618) [reply nrc_exporter.py:527]: Cool,I will change it right now.

**Comment by yihong0618** [comment]
@yasoob 
Done also change the Readme file.

**Comment by yasoob** [comment]
Thanks! Just merged it in.

**Comment by yihong0618** [comment]
> Thanks! Just merged it in.

Thank you~

**Pull request merged**